libs/blocks/ost/ost.js

```diff
@@ -1,24 +1,69 @@
-import { loadScript, loadStyle } from '../../utils/utils.js';
+import { getConfig, getLocale, loadScript, loadStyle } from '../../utils/utils.js';
+import { ENV_PROD, getTacocatEnv, getTacocatMetadata } from '../merch/merch.js';
 
 const IMS_COMMERCE_CLIENT_ID = 'aos_milo_commerce';
 const IMS_PROD_URL = 'https://auth.services.adobe.com/imslib/imslib.min.js';
 const OST_SCRIPT_URL = 'https://www.stage.adobe.com/special/tacocat/ost/lib/1.10.0/index.js';
 const OST_STYLE_URL = 'https://www.stage.adobe.com/special/tacocat/ost/lib/1.10.0/index.css';
 
-const ENVIRONMENT = 'PROD';
```

Comment:
@vladen OST should open always in PROD and not resolve the environment via the config.
I see no value in using Milo OST via stage env.
We can always test OST against stage though.

Reply:
@yesil OST always opens in prod now

- https://github.com/adobecom/milo/pull/771/files#diff-b8a8e71badbe17831d377dd74d796fa3e1e98ab182354e62bb86364fd01f1720R39
- https://github.com/adobecom/milo/pull/771/files#diff-01476d99c8ba4d00b7e767d0afbc424902d0d40e75c4ec4272a966edd0d4124dR4